### ISSUE TITLE
ignite failing micro service gateway build 

### ISSUE THREAD

**Issue by grajuu** [ruddell/ignite-jhipster#94]
Referring to @mraible **`Java Microservices with Spring Cloud Config and JHipster `** project 

For creating react native with ignite 
`ignite  new blog --boilerplate ignite-jhipster`

`Enter the path to your JHipster project root directory:`
which path should i provide ..? gateway ,blog or store ..?

I provided path as `gateway` 
it created react-native app perfectly but the build with ` mvn -Pprod verify com.google.cloud.tools:jib-maven-plugin:dockerBuild`. got failed at gateway itself.
with completion time.

```
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /Users/mardubudi/Documents/PBIS-REPO/tis-jhip/jhip-microservice/gateway/src/main/java/com/okta/developer/gateway/config/OAuth2SsoConfiguration.java:[6,69] cannot find symbol
  symbol:   class EnableOAuth2Sso
  location: package org.springframework.boot.autoconfigure.security.oauth2.client
[ERROR] /Users/mardubudi/Documents/PBIS-REPO/tis-jhip/jhip-microservice/gateway/src/main/java/com/okta/developer/gateway/config/OAuth2SsoConfiguration.java:[13,50] cannot find symbol
  symbol:   class OAuth2ClientContext
  location: package org.springframework.security.oauth2.client
[ERROR] /Users/mardubudi/Documents/PBIS-REPO/tis-jhip/jhip-microservice/gateway/src/main/java/com/okta/developer/gateway/config/OAuth2SsoConfiguration.java:[14,50] cannot find symbol
  symbol:   class OAuth2RestTemplate
  location: package org.springframework.security.oauth2.client
[ERROR] /Users/mardubudi/Documents/PBIS-REPO/tis-jhip/jhip-microservice/gateway/src/main/java/com/okta/developer/gateway/config/OAuth2SsoConfiguration.java:[15,59] package org.springframework.security.oauth2.client.resource does not exist
[ERROR] /Users/mardubudi/Documents/PBIS-REPO/tis-jhip/jhip-microservice/gateway/src/main/java/com/okta/developer/gateway/config/OAuth2SsoConfiguration.java:[23,2] cannot find symbol
  symbol: class EnableOAuth2Sso
[ERROR] /Users/mardubudi/Documents/PBIS-REPO/tis-jhip/jhip-microservice/gateway/src/main/java/com/okta/developer/gateway/config/OAuth2SsoConfiguration.java:[82,50] cannot find symbol
  symbol:   class OAuth2ProtectedResourceDetails
  location: class com.okta.developer.gateway.config.OAuth2SsoConfiguration
[ERROR] /Users/mardubudi/Documents/PBIS-REPO/tis-jhip/jhip-microservice/gateway/src/main/java/com/okta/developer/gateway/config/OAuth2SsoConfiguration.java:[83,50] cannot find symbol
  symbol:   class OAuth2ClientContext
  location: class com.okta.developer.gateway.config.OAuth2SsoConfiguration
[ERROR] /Users/mardubudi/Documents/PBIS-REPO/tis-jhip/jhip-microservice/gateway/src/main/java/com/okta/developer/gateway/config/OAuth2SsoConfiguration.java:[82,12] cannot find symbol
  symbol:   class OAuth2RestTemplate
  location: class com.okta.developer.gateway.config.OAuth2SsoConfiguration
[INFO] 8 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Gateway ............................................ FAILURE [01:35 min]
[INFO] Blog ............................................... SKIPPED
[INFO] Store .............................................. SKIPPED
[INFO] jhipster-parent .................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------- 
```
can you please help this .
--Thanks

**Comment by mraible** [comment]
It looks like you're using an old version of JHipster. You should generate your gateway with the latest version - v6.8.0.

**Comment by grajuu** [comment]
Not really, am on the latest,

```
$ jhipster --version
INFO! Using JHipster version installed globally
6.8.0
```

More over 
`Enter the path to your JHipster project root directory:`
which path should i provide ..? gateway ,blog or store ..?

**Comment by ruddell** [comment]
Just tested and you can remove the file causing the failed build and the app should work as expected.

`rm gateway/src/main/java/com/okta/developer/gateway/config/OAuth2SsoConfiguration.java`

I'll make a new release soon so this will no longer be an issue.

**Comment by grajuu** [comment]
Thanks @ruddell , 
 will that effect OAuth2 authentication with keycloak ,for  both react-native and web..?

**Comment by ruddell** [comment]
That file was added by `ignite-jhipster` so shouldn't affect the website, I tested the generated `ignite-jhipster` app with Keycloak and Okta and it worked fine without it.

**Comment by grajuu** [comment]
Sure thanks for the help.

**Comment by ruddell** [comment]
Fixed in https://github.com/ruddell/ignite-jhipster/commit/3a2adc273ec16d97bd9f77fca0441f8a26916fda and released in [v3.5.3](https://github.com/ruddell/ignite-jhipster/releases/tag/v3.5.3), thank you very much for reporting the issue.

Please continue opening issues if you find anything else broken.

**Comment by grajuu** [comment]
I am glad to do this as a fellow programmer, am happy to contribute some thing like from my side.
--Thanks.
